### PR TITLE
Change default server

### PR DESCRIPTION
**Pull request by BurakDev**

### CODE REVIEW

File: simp_le.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
 VERSION = '0'
 URL = 'https://github.com/kuba/simp_le'
 
-LE_STAGING_URI = 'https://acme-staging.api.letsencrypt.org/directory'
+LE_STAGING_URI = 'https://acme-v01.api.letsencrypt.org/directory'
```
Comment (kuba) [inline simp_le.py]: This constant has intentionally `STAGING` in name... please create a new constant and reference in the the argument setup code.

Comment (BurakDev) [inline simp_le.py]: is okay now ? or you want keep old constant

Comment (kuba) [inline simp_le.py]: Hmm, in my previous comment I explicitly said `create a new constant`, but after thinking about it for a while: `simp_le` is not a library - we don't need to store unnecessary/unused stuff here.

It's good as it is. Thanks.

**Comment by kuba** [comment]
LGTM, merging on green Travis.

**Pull request merged**